content/docs/user-guide/external-dependencies.md

```diff
@@ -64,6 +65,18 @@ $ dvc run -d s3://mybucket/data.txt \
           aws s3 cp s3://mybucket/data.txt data.txt
 ```
 
+### Microsoft Azure Blob Storage
+
+```dvc
+$ dvc run -d azure://my-container-name/data.txt \
+          -o data.txt \
+          az storage copy \
+                            -d data.json \
+                            --source-account-name my-account \
+                            --source-container my-container-name \
+                            --source-blob data.txt
```

Comment:
The indentation here seems excessive? Code block lines shouldn't go over 72 characters long, please. How about just align the flags under `copy`? Or go further back if still they go over 72 🙂

Reply:
The indentation is based on the indentation used in the HDFS example. I completely agree that it is excessive, but I figured there was probably a reason for it. I'll move it back a bit so they line up better with `copy`, I did check though and the longest line was 67 chars so it should be fine regardless

Reply:
Thanks for bringing this to out attention @steffansluis I'll review this in a separate PR 🙂